main/webservices/registration.soap.php

```diff
@@ -2862,23 +2915,6 @@ function WSCreateCourse($params)
             $course_language = $course_param['course_language'];
         }
 
-        // Set default values
-        if (isset($_user['language']) && $_user['language'] != '') {
-            $values['course_language'] = $_user['language'];
-        } else {
-            $values['course_language'] = api_get_setting('platformLanguage');
-        }
-
-        if (isset($_user['firstName'])) {
-            $values['tutor_name'] = api_get_person_name(
-                $_user['firstName'],
-                $_user['lastName'],
-                null,
-                null,
-                $values['course_language']
-            );
-        }
```

Comment:
I think this block is unneeded, because array $values is not used in the method WSCreateCourse. The tutor_name is set by the parameter sent to the webservice as well as the course_language.